main.go

```diff
@@ -45,7 +46,7 @@ const (
 
 func main() {
 	var cliConfig string
-	var APIGroup string
+	var APIGroups []string
```

Comment:
Can we rename the variables internally to `apiGroupVersions` to keep this inline with other K8s conventions?